src/hypergraph/viz/__init__.py

```
@@ -0,0 +1,34 @@
+)
+
+__all__ = [
+    "visualize",
+    "ScrollablePipelineWidget",
+    "VizDebugger",
+    "validate_graph",
+    "find_issues",
+    "extract_debug_data",
+    "RenderedDebugData",
+    "RenderedEdge",
+]
```

Ruff (0.14.14)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)